dandi/cli/cmd_validate.py

```diff
@@ -10,13 +10,11 @@
 @devel_option(
     "--schema", help="Validate against new BIDS schema version", metavar="VERSION"
 )
-@click.option("--report", help="Specify path to write a report under.")
 @click.option(
-    "--report-flag",
+    "--report",
     "-r",
     is_flag=True,
-    help="Whether to write a report under a unique path in the current directory. "
-    "Only usable if `--report` is not already used.",
+    help="Whether to write a report under a unique path in the current directory. ",
 )
```

Comment:
so we are breaking CLI ... why not just to keep `--report` to be an option to provide the path and if not specified, assume that no report writing was requested - print to the screen?  
Note: I might not be even able to write to current directory (dataset might be owned by someone else).

Reply:
functionality wasn't broken, help text was just incorrect.